internal-logging/src/main/java/co/elastic/otel/logging/AgentLog.java

```diff
@@ -47,6 +82,27 @@ public static void init() {
     Configurator.initialize(conf.build(false));
   }
 
+  public static void addSpanLoggingIfRequired(
+      SdkTracerProviderBuilder providerBuilder, ConfigProperties config) {
+
+    // Replicate behavior of the upstream agent: span logging exporter is automatically added when
+    // not already present when debugging. When logging exporter has been explicitly configured,
+    // spans logging will be done by the explicitly configured logging exporter instance.
+
+    logPlainText = config.getBoolean(OTEL_JAVAAGENT_DEBUG, false);
+    String exporterName = logPlainText ? "logging" : "otlp-logging";
+
+    boolean loggingExporterNotAlreadyConfigured =
+        !config.getList("otel.traces.exporter", emptyList()).contains(exporterName);
```

Comment:
if debug is off, `exporterName` is `otlp-logging`. Then if `otel.traces.exporter` includes `logging` (not `otlp-logging`) then the otlp logger is added, so we end up with both loggers. That actually seems fine to me, but I wanted to confirm that's what you intended